components/memix/actions/random/random.mjs

```diff
@@ -0,0 +1,20 @@
+import app from "../../memix.app.mjs";
+export default {
+  name: "Caption to Memix URL",
+  description:
+    "Generate a Memix share URL with a random template using the supplied caption. [See the docs here](https://api.memix.com/docs)",
+  key: "memix-random",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    caption: {
+      type: "string",
+      label: "Caption",
+    },
+  },
+  async run() {
```

Comment:
You could declare the `run` function with the $ parameter like:
`async run({ $ }) {` for later usage